release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3938,4 +3938,23 @@ $ oc adm release info 4.11.57 --pullspecs
 [id="ocp-4-11-57-upgrading"]
 ==== Updating
 
+To update a {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Comment:
Article here
```suggestion
To update an {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Reply:
You are right about this, I'll make the change to the other release notes files as well.